### PR TITLE
Private streaming channels

### PR DESCRIPTION
**Pull request by yoda7833**
Added the privateroom command with voice, purge, invite, kick, delete, and superdelete sub-commands.

TL;DR of commands:
Allows someone with the streaming activity in discord to create a voice channel and role that locks people out of the channel if they are missing the role. The creator of the channel can give or take the role away from people through the invite and kick commands.

private can be as a shortcut for privateroom
**!private create**: Creates a private voice channel if the author has the streaming activity on discord
**!private invite [member]**: Gives the author's private channel role to the member. The command will try to match a name or uses a mention.
**!private kick:** Removes the author's private channel role from the member. The command will try to match a name or uses a mention.
**!private delete**: Deletes the author's private voice channel
**!private superdelete [member]**: [must be admin] Deletes the members private channel
**!private purge**: [must be admin] Deletes all of the private channels

Changed the !wtf and !superwtf commands to accommodate for the sub commands

### CODE REVIEW

**Comment by yoda7833** [comment]
This doesn't use the database because I wasn't sure how fast that would be. So currently you need to kill the bot with **!die** if you don't want to clean up the private voice channels manually

**Comment by davidecotten** [comment]
I am going to hold off on this one until I replace the #promote_a_stream channel since the Waifubot components have stream detection functions.

**Comment by yoda7833** [comment]
Cool. If thats the case I'll probably go and make some changes that I was going to implement on my old bot that I never got around to before I brought them over to waifu bot
